app/templates/document_download_template.html

```diff
@@ -10,10 +10,18 @@
 {% endblock %}
 
 {% block head %}
-  <link rel="stylesheet" media="screen" href="{{ asset_url('stylesheets/main.css') }}" />
+  <!--[if !IE 8]><!-->
+    <link rel="stylesheet" media="screen" href="{{ asset_url('stylesheets/main.css') }}" />
+  <!--<![endif]-->
+  <!--[if IE 8]>
+    <link rel="stylesheet" href="{{ asset_url('stylesheets/main-ie8.css') }}">
```

Comment:
No `media=screen` attribute here intentional?

Reply:
I copied it from the [GOVUK Frontend docs](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/supporting-internet-explorer-8.md#including-the-ie8-stylesheet-in-your-project) but yes, it should match our other tags.